### PR TITLE
Fixed header wrapper

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Fixed header container: #1022

Now the header is normally located
90282778-a17cb300-de77-11ea-9b9e-04ffee303ab7

### CODE REVIEW

File: src/app/component/layout/components/header/header.component.scss [hunk 1]
```diff
@@ -42,7 +40,6 @@ $screen-320: " screen and (max-width : 320px)";
       width: 136px;
       height: 36px;
       margin-right: 4%;
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/header/header.component.scss:42]: I know that isn't your code but we shouldn't mix up the  `%` and `px`.

File: src/app/component/layout/components/header/header.component.scss [hunk 2]
```diff
@@ -487,11 +481,24 @@ $screen-320: " screen and (max-width : 320px)";
   }
 }
 
+@media screen and (max-width: 767px) {
+  .main {
+    .main-container {
+      margin: 0 1.95%;
```
Comment (mr-pavlenko) [inline src/app/component/layout/components/header/header.component.scss:487]: why is 1.95 %?

Comment (RomanDeveloperAcc) [reply src/app/component/layout/components/header/header.component.scss:487]: > why is 1.95 %?

According to the mockup, margin is 15 px at the screen width of 768px. 15*100/768 is close to 1.95.

File: src/app/component/layout/components/header/header.component.scss (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/component/layout/components/header/header.component.scss]: > I know that isn't your code but we shouldn't mix up the `%` and `px`.

Fixed: changed to pixels

**Pull request merged**